dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/ScheduleMapper.java

```diff
@@ -67,6 +72,12 @@
      * @param processDefinitionCode processDefinitionCode
      * @return schedule list
      */
+    @Cacheable(sync = true)
     List<Schedule> queryReleaseSchedulerListByProcessDefinitionCode(@Param("processDefinitionCode") long processDefinitionCode);
 
+    @CacheEvict(key = "#entity.processDefinitionCode")
```

Comment:
SpringEL is better, but now the `CacheEvictAspect.java ` can not support it. So can you try to solve SpringEL support?